src/main/java/dev/j3fftw/litexpansion/ItemSetup.java

```diff
@@ -21,12 +21,16 @@
 
 import javax.annotation.Nonnull;
 
+import static org.bukkit.Bukkit.getServer;
```

Comment:
No

Reply:
How do I fix this?

Reply:
Once again, use the actual Class.method() rather than doing a stupid static import.